frontend/src/scenes/billing/BillingProduct.tsx

```diff
@@ -60,6 +64,42 @@ export const BillingProductAddon = ({ addon }: { addon: BillingProductV2AddonTyp
         tierDisplayOptions.push({ label: `Current bill`, value: 'total' })
     }
 
+    const showPipelineAddonNotice =
+        addon.type === 'data_pipelines' &&
+        addon.subscribed &&
+        featureFlags['data-pipelines-notice'] &&
+        addon.plans?.[0].plan_key === 'addon-20240103-og-customers'
+
+    if (showPipelineAddonNotice) {
+        setProductSpecificAlert({
+            status: 'info',
+            title: 'Welcome to the data pipelines addon!',
+            message: `We've moved data export features (and cost) to the data pipelines addon to better reflect user needs. Your overall
+                    price hasn't changed.`,
```

Comment:
```suggestion
            title: 'Welcome to the data pipelines addon!',
            message: `We've moved data export features (and cost) here to better reflect user needs. Your overall
                    price hasn't changed.`,
```

This just helps avoid repeating the word data so much